src/store/reducers/cohorts.reducer.js

```diff
@@ -1,14 +1,44 @@
-import { COHORTS } from '../actions/actions.type';
+import { COHORTS, SINGLE_COHORT } from '../actions/actions.type';
```

Comment:
for the reducer, I also think that it should be a separated file for SINGLE_COHORT.

The INITIAL_DATA for single cohort is probably something like the following:
```
const initialData = {
    isLoading: false,
    isError: false
}
```
and the reducer will add data for each cohort when it is loaded.